src/components/Map/Map.tsx

```diff
@@ -229,8 +230,12 @@ class Map extends React.Component<MapProps> {
 
   render(): JSX.Element {
     return (
-      <Box height="500px">
-        <div ref={this.mapRef} style={{height: '100%'}} />
+      <Box height={['450px', null, '80vh', null]}>
+        {this.props.isMobile ? (
+          <div ref={this.mapRef} style={{height: '90%'}} />
```

Comment:
Do we want to use `Box` as opposed to `div` just for Chakra's sake or does this require an actual `div` object?